app/services/manifest_signer.rb

```diff
@@ -0,0 +1,46 @@
+class ManifestSigner
+  def initialize(args)
+    @args = args
+  end
+
+  def perform
+    cdx_start = Time.current
+
+    cdx_response = CDX::Manifest.new(parsed_args).sign
+
+    cdx_stop = Time.current
+    Rails.logger.debug(ANSI.blue{ "  CDX signature time: #{sprintf('%#g', (cdx_stop - cdx_start))} seconds" })
```

Comment:
should we add this kind of logging for everything or remove it from here? seems weird to have it in one CDX API caller class but not the other